api/server/controllers/agents/run.js

```diff
@@ -50,6 +50,25 @@ async function createRun({
     agent.model_parameters,
   );
 
+  /** Add proxy configuration for Anthropic */
+  if (provider === Providers.ANTHROPIC) {
+    const { ANTHROPIC_REVERSE_PROXY, ANTHROPIC_API_KEY } = process.env;
+
+    if (ANTHROPIC_API_KEY && ANTHROPIC_API_KEY !== 'user_provided' && !llmConfig.apiKey) {
+      llmConfig.apiKey = ANTHROPIC_API_KEY;
+    }
+
+    if (ANTHROPIC_REVERSE_PROXY) {
+      if (!llmConfig.clientOptions) {
+        llmConfig.clientOptions = {};
+      }
+
+      const baseURL = ANTHROPIC_REVERSE_PROXY.replace(/\/v1\/?$/, '');
```

Comment:
Why remove `/v1` specifically? is this a common pattern in reverse proxy URL?

Reply:
Yeah, good point. This might be relevant to just our proxy.